### DISCUSSION TITLE
Country Field not editable?

### DISCUSSION THREAD

**Discussion by santiagoGPNC** [photoprism/photoprism#3848] [Q&A] [answered]
Hello,
Photoprism identified some photos from Toulouse, France as Florida, USA (even though they have proper GPS coordinates on them), when I went to edit them  to fix it, I realised that the Country field isset to USA and it is not editable.

Any idea on how I can fix this?

**Comment by graciousgrey** [comment]
Hey :wave: 

If latitude and longitude are present, the country is automatically set based on the coordinates. To edit the country manually, you must first remove the latitude and longitude. However, this should not be necessary as the country should match the coordinates.

**Reply by santiagoGPNC** [reply]
hi!
maybe it's a bug? coordinates are totally correct. The photo was tagged as "Cape Coral, Florida", I edited out most of those tags but the country remains.
The coordinates point to this point https://www.google.com/maps/place/43°35'59.2%22N+1°26'56.4%22E/@43.59979,1.4464197,17z/data=!3m1!4b1!4m4!3m3!8m2!3d43.59979!4d1.449?entry=ttu
<img width="1269" alt="image" src="https://github.com/photoprism/photoprism/assets/143808451/549240c2-d15b-4007-8c7e-3dbdd519148f">

**Reply by graciousgrey** [reply, marked as answer]
Thanks for sharing the coordinates. Indeed, something seems to be wrong with the lookup of these coordinates, I can reproduce the problem. We will investigate this further.

**Reply by santiagoGPNC** [reply]
Thanks. I appreciate it.
if it is useful, other pictures in the same area were tagged in Florida also, but not in the same city. 
e.g. 43.604378 , 1.4436444 was tagged "Medley, Florida"

But others were tagged properly. e.g This one very near my first example was tagged properly
43.59978 , 1.4504528

**Reply by lastzero** [reply]
@santiagoGPNC Thank you for reporting this to us! I will look into it as soon as possible. Do you have any other pictures with incorrect location details?

**Reply by santiagoGPNC** [reply]
@lastzero  not that many. it's around 8 photos. most of them from in the same location so they all got the same mistake (1 for Cape Coral, the rest for Medley, Florida. All of them taken at Toulouse ).
But Im adding a lot of new photos, so I'll report additional errors.

**Reply by lastzero** [reply]
@santiagoGPNC That's very helpful! It would be great if you could then create a formal bug report for this issue, so we can track it:

https://github.com/photoprism/photoprism/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=Geocoding:%20Incorrect%20location%20details%20for%20some%20coordinates

Information about your server hardware etc. is not required in this case. Since we can fix this in our reverse geocoding backend, there is probably no need to release an update for it either.

**Reply by santiagoGPNC** [reply]
thanks @lastzero  I've done so. https://github.com/photoprism/photoprism/issues/3849

**Comment by santiagoGPNC** [comment]
Since it's been suggested this is a bug, I'll close this question.
The bug I opened: 
https://github.com/photoprism/photoprism/issues/3849
